Misc/NEWS.d/next/Windows/2019-02-02-22-12-23.bpo-35890.ccIjHH.rst

```diff
@@ -0,0 +1 @@
+Call unicode version GetVersionExW in Python/sysmodule.c, and add a WCSTOK macro to Python/pathconfig.c to wrap the wcstok_s.
```

Comment:
This appears in the summary for users, so I'd suggest putting this line in the commit message and something more like the PR title here.

Reply:
Commit message and news.d updated.